delegatedadmin.adoc

```diff
@@ -97,18 +96,18 @@ Zimlet entry
 == Rights
 
 Rights are the functions that a delegated administrator can or cannot
-perform on a named target. Right can be either system-defined or attribute.
+perform on a named target. A right can be either system-defined or attribute.
```

Comment:
Proposed change:

Rights are the functions that a delegated administrator can or cannot
perform on a named target. A right is either system-defined or granted
at the attribute level.